src/tracer.js

```diff
@@ -29,6 +35,21 @@ export const ASYNC_HANDLER_REJECTED = 'async_handler_rejected';
 export const NON_ASYNC_HANDLER_ERRORED = 'non_async_errored';
 export const LEAK_MESSAGE =
   'Execution leak detected. More information is available in: https://docs.lumigo.io/docs/execution-leak-detected';
+const TIMEOUT_BUFFER_MS = 500;
```

Comment:
half a second is quite a long time, why pick such a high number?

Reply:
We going with `250ms`, maybe we will reduce this later